Doc/library/asyncio-task.rst

```diff
@@ -335,6 +335,10 @@ and reliable way to wait for all tasks in the group to finish.
       Create a task in this task group.
       The signature matches that of :func:`asyncio.create_task`.
 
+      .. versionchanged:: 3.13
+
+         Close the given coroutine if the task group is not active.
+
```

Comment:
I believe that the convention is that a description of the new/changed behavior is added to the main text of the documentation (in this case the two lines right above) and then the `versionchanged` directive is used to clarify when this behavior was added or changed. In this case that would be a little redundant but I still feel it's appropriate.

For example, the main text could include information about when a task group is considered inactive (e.g. not yet entered, already finished, or in the process of shutting down).

Reply:
Thanks for the explanation! I've updated the main text